frontend/src/app/modules/interventions/map/map.component.ts

```diff
@@ -18,13 +20,19 @@ export class MapComponent {
   interventionMapGtmContext: string;
   loading$ = forkJoin(
     this.tilesLoadedSubject.asObservable().pipe(first()),
-    this.interventionsService.getInterventions().pipe(tap(data => (this.interventions = data))),
+    this.interventionsService
+      .getInterventions({ Status: InterventionStatus.ActionRequired })
+      .pipe(tap(data => (this.interventions = data))),
```

Comment:
Not related to your changes directly, but I found a bug on map view when testing your changes.
When returned `data` is empty array `[]`, error gets thrown to console. 

The map expects at least one element since the hackathon days. If you could come up with something simple to fix it then we can do it in this PR - if not, let's keep this bug for later. Maybe we should try to keep the last location of map or something :)

Reply:
I'd leave it for a separate PR - this PR is already some time in review.